packages/core/src/index.ts

```diff
@@ -213,6 +213,9 @@ class Hopr<Chain extends HoprCoreConnector> extends EventEmitter {
         relay: {
           enabled: false
         }
+      },
+      dialer: {
+        maxParallelDials: 1000
```

Comment:
Can we flag this only to be available to `bootstrap` servers?

Reply:
ok